get_rhcos_image.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -xe
+
+source common.sh
+
+RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-47.145}"
+RHCOS_IMAGE_NAME="redhat-coreos-maipo-${RHCOS_IMAGE_VERSION}"
+RHCOS_IMAGE_FILENAME="${RHCOS_IMAGE_NAME}-openstack.qcow2"
+if [ ! -f "$RHCOS_IMAGE_FILENAME" ]; then
+    curl --compressed -L -O "https://releases-redhat-coreos-dev.cloud.paas.upshift.redhat.com/storage/releases/maipo/${RHCOS_IMAGE_VERSION}/${RHCOS_IMAGE_FILENAME}"
```

Comment:
This needs a rebase since the curl was broken due to the non-root user patch landing which removed the --insecure option (or perhaps we need to install the certs onto the test boxes?)

Reply:
rebased